diff -Nuard evolution-sharp-0.3.orig/configure.in evolution-sharp-0.3/configure.in
--- evolution-sharp-0.3.orig/configure.in	2004-06-02 02:25:33.000000000 +0200
+++ evolution-sharp-0.3/configure.in	2004-07-05 20:44:12.892756504 +0200
@@ -13,7 +13,23 @@
 
 PKG_CHECK_MODULES(BASE_DEPENDENCIES, evolution-data-server-1.0 >= 0.0.93)
 
-PKG_CHECK_MODULES(MONO, mono >= 0.91)
+MONO_REQUIRED_VERSION=0.91
+PKG_CHECK_MODULES(MONO, mono >= $MONO_REQUIRED_VERSION, has_mono=true, has_mono=false)
+PKG_CHECK_MODULES(MINT, mint >= $MONO_REQUIRED_VERSION, has_mint=true, has_mint=false)
+ 
+if test "x$has_mono" = "xtrue"; then
+    echo "using mono"
+    RUNTIME="mono --debug"
+else
+    if test "x$has_mint" = "xtrue"; then
+        echo "using mint"
+	RUNTIME="mint"
+    else
+	AC_MSG_ERROR([Unable to find mono or mint packages]);
+    fi
+fi
+AC_SUBST(RUNTIME)
+
 AC_PATH_PROG(CSC, mcs, no)
 if test "x$CSC" = "xno"; then
 	AC_MSG_ERROR([Your mono installation doesn't expose mcs])
@@ -22,6 +38,8 @@
 if test "x$GACUTIL" = "xno"; then
 	AC_MSG_ERROR([Your mono installation doesn't expose gacutil])
 fi
+
+
 if test `uname -s` = "Darwin"; then
         LIB_PREFIX=
         LIB_SUFFIX=.dylib
diff -Nuard evolution-sharp-0.3.orig/evolution-sharp.pc.in evolution-sharp-0.3/evolution-sharp.pc.in
--- evolution-sharp-0.3.orig/evolution-sharp.pc.in	2004-06-02 02:53:41.000000000 +0200
+++ evolution-sharp-0.3/evolution-sharp.pc.in	2004-07-05 20:44:59.318698688 +0200
@@ -1,5 +1,5 @@
 prefix=@prefix@
-libdir=${prefix}/lib
+libdir=@libdir@
 
 Name: Evolution#
 Description: Evolution# - Evolution .NET Binding
